main.py

```diff
@@ -238,6 +239,9 @@ def register_toolkit_for_master_organisation():
     # AgentWorkflowSeed.doc_search_and_code(session)
     # AgentWorkflowSeed.build_research_email_workflow(session)
     replace_old_iteration_workflows(session)
+    
+    if env == "DEV":
+        install_local_llms(session)
```

Comment:
This is where the local LLM records are added to the database so they show up in the drop down. Currently its just ooba